discovery/source/DiscoveryClient.cpp

```diff
@@ -21,15 +21,25 @@ namespace Aws
 
         DiscoveryClient::DiscoveryClient(
             const Aws::Discovery::DiscoveryClientConfig &clientConfig,
-            Crt::Allocator *allocator) noexcept
+            Crt::Allocator *allocator,
+            Crt::String ggServerName) noexcept
```

Comment:
Shouldn't the `ggServerName` be a member of `DiscoveryClientConfig`, instead of a new constructor arg? The purpose behind have a "config" struct is to let us easily add additional args like this

Reply:
moved to config